parsec/_parsec_pyi/enumerate.pyi

```diff
@@ -58,8 +58,8 @@ class RealmRole:
     MANAGER: RealmRole
     CONTRIBUTOR: RealmRole
     READER: RealmRole
-    @classmethod
-    def values(cls) -> list[RealmRole]: ...
+    values: list[RealmRole]
```

Comment:
If value is a constant, we should rename it to `VALUES` and ideally make it a tuple